minimum-constraints.txt

```diff
@@ -86,7 +86,7 @@ wheel==0.33.5; python_version >= '3.8'
 
 # Direct dependencies for installation (must be consistent with requirements.txt)
 
-mock==2.0.0
+mock>=2.0.0; python_version < "3.3"
```

Comment:
This should remain a '==' since this is a minimum constraints file. I'll fix that.

Reply:
DONE in PR #2594.